website/content/docs/concepts/credential-management.mdx

```diff
@@ -99,7 +99,9 @@ Credential injection is required for the SSH target type, allowing users to inje
 - Usernames and passwords
 - Usernames and public keys
 
-Additional credentials can be brokered to SSH targets after the session is establised using injected credentials.
+The SSH server must be configured to allow password authentication. Keyboard-interactive authentication is not supported for credential injection.
```

Comment:
In the same vein as Louis' suggestion, we should make it clear that `password` authentication is necessary when using `username/password` as the injected credential, not otherwise.

Reply:
Thanks for the clarification! I inverted the phrasing and specifically called out Username password credentials.